[2025-05-11T06:27:24.150+0700] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-05-11T06:27:24.202+0700] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: bbca_lstm_prediction_enhanced.hyperparameter_tuning scheduled__2025-05-08T11:30:00+00:00 [queued]>
[2025-05-11T06:27:24.212+0700] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: bbca_lstm_prediction_enhanced.hyperparameter_tuning scheduled__2025-05-08T11:30:00+00:00 [queued]>
[2025-05-11T06:27:24.213+0700] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-05-11T06:27:24.230+0700] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): hyperparameter_tuning> on 2025-05-08 11:30:00+00:00
[2025-05-11T06:27:24.238+0700] {standard_task_runner.py:63} INFO - Started process 7448 to run task
[2025-05-11T06:27:24.241+0700] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'bbca_lstm_prediction_enhanced', 'hyperparameter_tuning', 'scheduled__2025-05-08T11:30:00+00:00', '--job-id', '120', '--raw', '--subdir', 'DAGS_FOLDER/bbca_dag.py', '--cfg-path', '/tmp/tmpfyjgjax2']
[2025-05-11T06:27:24.244+0700] {standard_task_runner.py:91} INFO - Job 120: Subtask hyperparameter_tuning
[2025-05-11T06:27:24.328+0700] {task_command.py:426} INFO - Running <TaskInstance: bbca_lstm_prediction_enhanced.hyperparameter_tuning scheduled__2025-05-08T11:30:00+00:00 [running]> on host 63831b4a45c6
[2025-05-11T06:27:24.835+0700] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='bbca_lstm_prediction_enhanced' AIRFLOW_CTX_TASK_ID='hyperparameter_tuning' AIRFLOW_CTX_EXECUTION_DATE='2025-05-08T11:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-08T11:30:00+00:00'
[2025-05-11T06:27:24.837+0700] {taskinstance.py:430} INFO - ::endgroup::
[2025-05-11T06:27:24.876+0700] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-05-11T06:27:24.877+0700] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/bbca_dag.py", line 72, in task_hyperparameter_tuning
    params = hyperparameter_tuning(data_path, model_type)
             ^^^^^^^^^^^^^^^^^^^^^
NameError: name 'hyperparameter_tuning' is not defined. Did you mean: 'task_hyperparameter_tuning'?
[2025-05-11T06:27:24.913+0700] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=bbca_lstm_prediction_enhanced, task_id=hyperparameter_tuning, run_id=scheduled__2025-05-08T11:30:00+00:00, execution_date=20250508T113000, start_date=20250510T232724, end_date=20250510T232724
[2025-05-11T06:27:24.939+0700] {standard_task_runner.py:110} ERROR - Failed to execute job 120 for task hyperparameter_tuning (name 'hyperparameter_tuning' is not defined; 7448)
[2025-05-11T06:27:24.977+0700] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2025-05-11T06:27:25.019+0700] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-11T06:27:25.026+0700] {local_task_job_runner.py:222} INFO - ::endgroup::
